src/gui/src/assets/keyboard_mixin.js

```diff
@@ -83,12 +83,12 @@ const keyboardMixin = targetId => ({
             card.show = this.card_items[this.pos].querySelector(".card");
             card.id = this.card_items[this.pos].dataset.id;
             card.close = document.querySelector("[data-dialog='" + dialog + "-detail'] [data-btn='close']");
-            //card.close = document.querySelector(".v-dialog--active [data-btn='close']");
+            // Link is always extracted from the news item newdial button, does not work for opening all selected items, but also with opened items
+            card.link = this.card_items[this.pos].querySelector(".newdial button[data-btn='link'] a");
```

Comment:
this refers to an obsolete .newdial class and probably wouldn't work

Reply:
I have some more commits locally fixing that, will create a new PR with some stuff.